lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -334,10 +343,38 @@ def format(tag, time, record)
       [tag, time, record].to_msgpack
     end
 
+    # Convert tags.
+    # If 'require_valid_tags' is true, reject any non-string tags or tags with
```

Comment:
Changed to this except the first line (probably there by accident). The method name has been changed as well.